hack/ci/e2e.sh

```diff
@@ -187,9 +187,50 @@ EOF
         | grep -cv "node-role.kubernetes.io/master" \
     )"
 
+    KUBERNETES_PRESUBMIT_JOB=true
     # ginkgo regexes
-    SKIP="${SKIP:-}"
-    FOCUS="${FOCUS:-"\\[Conformance\\]"}"
+    if [[ "${KUBERNETES_PRESUBMIT_JOB:-}" = true ]]; then
+        # k/k presubmit jobs regex
+        SKIP="${SKIP:-"\\[Slow\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]"}"
+        FOCUS="${FOCUS:-""}"
+        # skip tests that are not able to rung with kind
+        # [sig-node] RuntimeClass should run a Pod requesting a RuntimeClass with a configured handler [NodeFeature:RuntimeHandler]
+        # https://github.com/kubernetes/kubernetes/issues/80098
+        SKIP="RuntimeClass.should.run.a.Pod.requesting.a.RuntimeClass.with.a.configured.handler|${SKIP}"
+        # [sig-storage] In-tree Volumes [Driver: nfs]
+        # [sig-storage] PersistentVolumes NFS
+        # TODO: kind nodes can't mount nfs volumes: install apt install -y nfs-common in the base image, kernel version, ...??
+        SKIP="In-tree.Volumes.\\[Driver:.nfs\\]|PersistentVolumes.NFS|${SKIP}"
```

Comment:
@BenTheElder this works for me in a kernel 5.2.2, the fix was in version 4.16.
Is it possible to run that kernel version in the CI?
We'll need to install nfs-common in the nodes too.

Reply:
no, the nodes are not going to be that new anytime soon. pretty sure they are ubuntu LTS or COS so